tests/main/component-refreshes/task.yaml

```diff
@@ -0,0 +1,40 @@
+summary: Test refreshing a component from the store
+
+details: |
+  Verifies that we can refresh a snap with components from the store.
+
+systems: [ubuntu-16.04-64, ubuntu-18.04-64, ubuntu-2*, ubuntu-core-*, fedora-*]
+
+restore: |
+  snap refresh --unhold
+  # TODO:COMPS: properly remove the snap and components
```

Comment:
I think that if not removed properly it will cause trouble to the next test. But the remove PRs are about to be merged, so once that happens this PR can be rebased and the clean-up code should do the right thing simply by removing the snaps.